zero_bin/tools/prove_rpc.sh

```diff
@@ -62,12 +62,12 @@ RECOMMENDED_FILE_HANDLE_LIMIT=8192
 
 mkdir -p $PROOF_OUTPUT_DIR
 
-if [ $IGNORE_PREVIOUS_PROOFS ]; then
+if $IGNORE_PREVIOUS_PROOFS ; then
```

Comment:
A detail, but please refrain to add this kind of changes unrelated to the PR (there are quite a few in this PR), for reviewers' sake